analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -71,12 +71,6 @@ import org.ossreviewtoolkit.utils.common.withoutPrefix
 import org.ossreviewtoolkit.utils.core.log
 import org.ossreviewtoolkit.utils.spdx.SpdxConstants
 
-object NpmCli : CommandLineTool {
```

Comment:
Not sure whether it makes a big difference, but you could keep the objects implementing `CommandLineTool`, and instead of extending the `CommandLineTool` interface in `Npm`, introduce a function (or property) like `getCommandLineTool()` that returns the tool for this concrete implementation.

Reply:
In the current architecture, it's actually more convenient if really the package manager classes themselves implement the `CommandLineTool` interface, as they then show up in the correct section for `ort requirements`. Also see my remark in [this commit message](https://github.com/oss-review-toolkit/ort/pull/5162/commits/0c8cd71e6ef741a3816fd17fb165c55ad27b5bec).